### ISSUE TITLE
Error listing templates on Windows

### ISSUE THREAD

**Issue by c4s4** [c4s4/neon#8]
Neon crashes when listing templates on Windows:

```bash
$ ./neon.exe -templates
panic: stating file: CreateFile c4s4/build/build.tpl: The system cannot find the path specified.

goroutine 1 [running]:
neon/build.PrintTemplates(0x7e347c, 0x7)
        /home/casa/doc/perso/neon/src/neon/build/repo.go:201 +0xfc
main.printHelp(0x10000, 0x0, 0x0, 0x0, 0x0, 0x0, 0x7e347c, 0x7, 0x11e50000)
        /home/casa/doc/perso/neon/src/neon/main.go:235 +0xc5
main.main()
        /home/casa/doc/perso/neon/src/neon/main.go:169 +0x2bc
```

**Comment by c4s4** [comment]
This happens also on Linux.

**Comment by c4s4** [comment]
Fixed in release *1.0.0-rc-7*.
